test/nightly/deploy.test.ts

```
@@ -59,10 +59,21 @@ async function validateWebSite(webUrl: string | undefined, client: ServiceClient
     let response: HttpOperationResponse;
     // eslint-disable-next-line no-constant-condition
     while (true) {
-        response = await client.sendRequest({ method: 'GET', url: webUrl });
-        if (Date.now() > endTime || response.status == 200) {
-            break;
+        try {
+            response = await client.sendRequest({ method: 'GET', url: webUrl });
+            if (Date.now() > endTime || response.status == 200) {
+                break;
+            }
+        } catch (error) {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            if (error?.statusCode === 404 && /websitedisabled/i.test(error?.body)) {
```

Honestly I'd probably just ignore all errors here. They're probably transient and as long as we get a 200 eventually that's all that really matters for the test